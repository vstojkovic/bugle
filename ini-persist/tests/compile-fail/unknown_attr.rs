use ini_persist::IniLoad;

#[derive(IniLoad)]
#[ini(bogus)]
struct Foo {
    argle: u8,
    bargle: String,
}

fn main() {
}
