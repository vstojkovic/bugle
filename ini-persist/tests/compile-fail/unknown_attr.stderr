error: unknown ini attribute `bogus`
 --> tests/compile-fail/unknown_attr.rs:4:7
  |
4 | #[ini(bogus)]
  |       ^^^^^
